app/components/edit/EditSidebar.scss

```diff
@@ -7,8 +7,12 @@
 	width: calc-em(250px);
 	background-color: $color-grey1;
   display: flex;
-	justify-content: space-between;
 	flex-direction: column;
+  position: sticky;
+  position: -webkit-sticky;
```

Comment:
You probably shouldn't be explicitly writing the vendor prefix properties here, since we really ought to be running a CSS postprocessor that outputs the complete set of vendor prefixes. If we don't have one set up yet, can we install and configure [Autoprefixer](https://github.com/postcss/autoprefixer) to add these prefixes?

Reply:
@richardxia I can set up the autoprefixer in a separate issue/PR? to keep the scope of this PR clear